src/anet.c

```diff
@@ -50,8 +50,7 @@
 #include "anet.h"
 #include "config.h"
 #include "util.h"
-
-#define UNUSED(x) (void)(x)
+#include "server.h"
```

Comment:
Oh.. I didn't realize this implication. To keep some parts less coupled with the huge server.h which essentially makes everything depend on everything, we usually try to avoid including it when possible. There is a smaller header file `serverassert.h` that provides an `assert` macro that we can use instead. It has the same effect, but it avoids the coupling.

Reply:
```suggestion
#include "serverassert.h"
```